x-pack/plugins/ml/public/application/components/ml_page/side_nav.tsx

```diff
@@ -154,7 +154,7 @@ export function useSideNavItems(activeRoute: MlRoute | undefined) {
           {
             id: 'anomaly_detection',
             name: i18n.translate('xpack.ml.navMenu.anomalyDetection.jobsManagementText', {
-              defaultMessage: 'Jobs management',
+              defaultMessage: 'Jobs',
```

Comment:
As @lcawl suggested, what did you decide about shortening the `Nodes Overview` title / link to the more concise `Nodes`?

Reply:
renamed in https://github.com/elastic/kibana/pull/124049/commits/bfe02d1198fa69eab33cf7e0f773c4a80856621c